packages/apputils-extension/src/index.tsx

```diff
@@ -25,11 +34,14 @@ import {
 import { extensionIcon } from '@jupyterlab/ui-components';
 
 import {
+  WorkspaceManager,
```

Comment:
perhaps `LiteWorkspaceManager` to be consistent with other divergences from upstream.